src/games/gameitem.py

```diff
@@ -218,99 +218,18 @@ def tooltip(self, data):
 
 
 class GameTooltipFormatter:
-    TIP_FORMAT = str(util.THEME.readfile("games/formatters/tool.qthtml"))
+
+    icon_path = os.path.join("chat", "countries/")
+    icon_abs_path = os.path.join(util.COMMON_DIR, icon_path)
```

Comment:
This stuff is used in just one function, we should move it there. In general class namespace should only contain class-specific constants and stuff like that.